### DISCUSSION TITLE
Hide Warning "does not have any default listener" in Logs

### DISCUSSION THREAD

**Discussion by muelleel** [Minestom/Minestom#1241] [Help] [answered]
Hi,

I've registered an PlayerPacketEvent to get the typed input from an AnvilGUI:
```
SOMEINSTANCE.eventNode().addListener(PlayerPacketEvent.class, event -> {
    if (event.getPacket() instanceof ClientNameItemPacket packet) {
        typedText = packet.itemName();
    }
});
```

This works fine, but the server console is spitting the following message for each character which is typed:

`[Ms-Tick-0] [18:37:17] (PacketListenerManager.processClientPacket) - WARN - Packet class net.minestom.server.network.packet.client.play.ClientNameItemPacket does not have any default listener! (The issue comes from Minestom)`

This is really spamming the logs when multiple users are typing a lot. 
Is there any way of disabling this type of message without forking the source code?

Minestom `ab5734334c`

Thanks :)

**Comment by dev-hydrogen** [comment, marked as answer]
You can add your own (dummy) listener, the method to register a listener is public: https://github.com/Minestom/Minestom/blob/master/src/main/java/net/minestom/server/listener/manager/PacketListenerManager.java

**Reply by muelleel** [reply]
Yes

`        
MinecraftServer.getPacketListenerManager().setListener(ClientNameItemPacket.class, (packet, player) -> {});
`

does the trick!
Thank you :+1:
